### DISCUSSION TITLE
ToggleButtons multiple not working

### DISCUSSION THREAD

**Discussion by eae-omeyer** [filamentphp/filament#12461] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.71

### How can we help you?

Hey,

I want to implement ToggleButtons (multiple) in a resource form, but can't get it working properly.
Just testing with the example from the documentation:

`ToggleButtons::make('technologies')
    ->multiple()
    ->options([
        'tailwind' => 'Tailwind CSS',
        'alpine' => 'Alpine.js',
        'laravel' => 'Laravel',
        'livewire' => 'Laravel Livewire',
    ])`

The buttons get rendered as expected, but I'm not able to select a single or multiple options.
When I click on any unselected option, all options are selected. When I click on any selected
option, all options are unselected. In practice, it's all or nothing.

No errors in the console. Assets are newest versions.

What am I missing here?

![image](https://github.com/filamentphp/filament/assets/104062629/423e0851-75f7-434b-a80d-449180b42be9)
![image](https://github.com/filamentphp/filament/assets/104062629/d519e24e-b690-4f3c-852b-2d7428329700)

**Comment by leandrocfe** [comment]
Could you share the whole code you are trying?

**Comment by eae-omeyer** [comment]
This is the form used in the resource:

![image](https://github.com/filamentphp/filament/assets/104062629/7f67d5ce-9976-4a71-b3ec-bc13afb6b5bd)

**Reply by leandrocfe** [reply]
bug, probably. Let me check it..

**Reply by eae-omeyer** [reply]
Thanks for digging into it!

**Reply by leandrocfe** [reply]
Is it happening in the EditPage or EditAction? Could you confirm if it works in the CreatePage/CreateAction?

**Reply by eae-omeyer** [reply]
Yes, it is happening in the edit page. And yes, it works in the create page - just tested.

**Reply by leandrocfe** [reply]
Could you open an issue please?

**Reply by eae-omeyer** [reply]
Yes, and thank you for your support!

**Reply by eae-omeyer** [reply]
My first github issue - Is it really necessary to create a github repository for reproducing this?

**Reply by leandrocfe** [reply]
Yes

**Reply by eae-omeyer** [reply]
Done -> https://github.com/filamentphp/filament/issues/12474

**Reply by eae-omeyer** [reply, marked as answer]
Resolved: The corresponding table column needs an empty array as default value - 'null' is not working.
